src/js/components/Marathon.jsx

```diff
@@ -178,6 +181,15 @@ var Marathon = React.createClass({
     );
   },
 
+  getChaosModal: function () {
+    return (
+      <ChaosModalComponent
+        isChaosInProgress={this.state.isChaosInProgress}
```

Comment:
For me this is only a nice-to-have and very expensive to integrate on Marathon side I think, because this would introduce a new API-endpoint to check the status.
For now all we are doing to produce chaos is to call the tasks/delete-endpoint from frontend side.

I don't think that this is necessary to know the tasks deletion status? And how should this look like?

We make chaos, so let's stick with chaos, haha!